backend/src/api/scans.ts

```diff
@@ -245,3 +250,25 @@ export const invokeScheduler = wrapHandler(async (event) => {
     body: ''
   };
 });
+
+export const runScan = wrapHandler(async (event) => {
+  await connectToDatabase();
+  const id = event.pathParameters?.scanId;
+  if (!id || !isUUID(id)) {
+    return NotFound;
+  }
+  const scan = await Scan.findOne({
+    id: id
+  });
+
+  if (scan) {
+    scan.lastRun = null;
```

Comment:
Instead of setting `lastRun` to null, we should instead create a new column specifically for this, perhaps called `manualRunPending`. That way, `lastRun` still keeps its meaning as "the date the scan was last run".

Reply:
Sounds good

Reply:
Actually, looking at it, I understand your point, but as soon as it's set to null, the scheduler will trigger a new scan and lastRun will be set to the correct time again. I can still change it though, if you'd prefer a separate column.

Reply:
additionally, the times of the previous finished scans are stored in the scan tasks, so the historical data is still preserved